File: tks/WebService/src/main/kotlin/webservice/dto/LibraryItemSoapDto.java
package webservice.dto;


import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class LibraryItemSoapDto {
    private String accessionNumber;
    private String title;
    private String author;
    private String publisher;
    private String type;
}

File: tks/WebService/src/main/kotlin/webservice/dto/LibraryItemSoapDto.kt
package webservice.dto

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlElement
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
open class LibraryItemSoapDto {
    @XmlElement(namespace = "http://superbiz.org/schema/1.0")
    var accessionNumber: String? = null
    @XmlElement(namespace = "http://superbiz.org/schema/1.0")
    var title: String? = null
    @XmlElement(namespace = "http://superbiz.org/schema/1.0")
    var author: String? = null
    @XmlElement(namespace = "http://superbiz.org/schema/1.0")
    var publisher: String? = null
    @XmlElement(namespace = "http://superbiz.org/schema/1.0")
    var type: String? = null
}